src/app/pages/dashboard/components/rekey-search-card/rekey-search-card.html

```diff
@@ -1,12 +1,12 @@
 <button clear (click)="navigateToRekeySearch()">
-  <ion-card class="rekey-search-card">
+  <ion-card class="rekey-search-card" id="rekey-search-card">
     <div id="card-content-wrapper">
       <ion-grid>
         <ion-row class="icon-row">
           <img src="assets/imgs/dashboard/search-rekey.png" alt="Search test bookings to rekey" class="icon-search-rekey-tests"/>
         </ion-row>
         <ion-row>
-          <ion-button class="mes-neutral-button">
+          <ion-button id="rekey-search" class="mes-neutral-button">
```

Comment:
call this "rekey-search-button" instead of just "rekey-search"

Reply:
done